website/docs/config/implicit-accounts.mdx

```diff
@@ -69,15 +69,36 @@ These accounts will be created and funded automatically when a sandbox starts. A
     "sandbox": {
         "local": {
             "accounts": {
+                "default": "bob",
                 "bob": {
                     "encryptedKey": "edpkurPsQ8eUApnLUJ9ZPDvu98E8VNj4KtJa1aZr16Cr5ow5VHKnz4",
                     "publicKeyHash": "tz1aSkwEot3L2kmUvcoxzjMomb9mvBNuzFK6",
-                    "secretKey": "unencrypted:edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt"
+                    "secretKey": "unencrypted:edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt",
```

Comment:
Just a note, that we'll need to come back to this file an update it when we add mnemonic seedphrases.